unittests/filedialog_test.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2017 CEF Python, see the Authors file.
```

Comment:
Can this test be added to main_test.py? Is it required to make it a separate test? This test duplicates much code from main_test.py.